dScripts/02_server/Map/Property/AG_Small/ZoneAgProperty.cpp

```diff
@@ -413,7 +413,7 @@ void ZoneAgProperty::BaseOnFireEventServerSide(Entity* self, Entity* sender, std
 		if (player == nullptr)
 			return;
 
-		player->GetCharacter()->SetPlayerFlag(self->GetVar<uint32_t>(defeatedProperyFlag), true);
+		player->GetCharacter()->SetPlayerFlag(self->GetVar<int32_t>(defeatedProperyFlag), true);
```

Comment:
If we're changing these to int32_t in this PR their setter in each respective property setter needs to be updated (right now these are stored as a uint32 since at the time we marked them as unsigned)